src/Mod/Measure/Gui/QuickMeasure.cpp

```diff
@@ -112,22 +112,22 @@ bool QuickMeasure::canMeasureSelection(const Gui::SelectionChanges& msg) const
     return doc != nullptr;
 }
 
+bool QuickMeasure::isObjAcceptable(App::DocumentObject* obj)
+{
+    if (!obj || !obj->isDerivedFrom(Part::Feature::getClassTypeId())) {
+        return false;
+    }
+
+    std::string vpType = obj->getViewProviderName();
+    auto* vp = Gui::Application::Instance->getViewProvider(obj);
+    return !(vpType == "SketcherGui::ViewProviderSketch" && vp->isEditing());
```

Comment:
It would make me happier if we explicitly checked `vp` before dereferencing it:
```suggestion
    return !(vpType == "SketcherGui::ViewProviderSketch" && vp && vp->isEditing());
```